fili-core/src/main/java/com/yahoo/bard/webservice/table/resolver/AggregatableDimensionsMatcher.java

```diff
@@ -43,7 +43,7 @@ public AggregatableDimensionsMatcher(DataApiRequest request, TemplateDruidQuery
 
     @Override
     public boolean test(PhysicalTable table) {
-        Set<String> columnNames = TableUtils.getColumnNames(request, query.getInnermostQuery(), table);
+        Set<String> columnNames = TableUtils.getColumnNames(request, query.getInnermostQuery());
```

Comment:
After looking into this.... I discovered that not only that the test was no sufficient, in addition... `AggregatableDimensionsMatcher` never worked the way it supposed to work but instead worked the opposite way it supposed to work =)